### PR TITLE
SDKS - 2031 Bugs fixed when change on client_id to load from persisted options

### PR DESCRIPTION
**Pull request by jeyanthanperiyasamy**
# JIRA Ticket

Please link jira ticket here

# Description

Briefly describe the change and any information that would help speedup the review and testing.

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

**Review by witrisna — APPROVED** [review]

File: forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt [hunk 1]
```diff
@@ -8,34 +8,27 @@
...
 
-         const val realm = "realm"
-         const val url = "url"
-         const val cookieName = "cookieName"
-         const val clientId = "client_id"
-         const val revokeEndpoint = "revoke_endpoint"
-         const val endSessionEndpoint = "end_session_endpoint"
-         const val scope = "scope"
-         const val redirectUri = "redirect_uri"
+        private  const val realm = "realm"
```
Comment (george-bafaloukas-forgerock) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt:20]: super minor, the `private  const val realm = "realm"` has two spaces before const

File: forgerock-auth/src/main/java/org/forgerock/android/auth/FROptions.kt [hunk 2]
```diff
@@ -147,9 +147,9 @@ data class OAuth(val oauthClientId: String? = "",
  * Oauth builder to build the SDK configuration information specific to oauth
  */
 class OAuthBuilder {
-    var oauthClientId: String? = ""
-    var oauthRedirectUri: String? = ""
-    var oauthScope: String? = ""
+    var oauthClientId: String = ""
```
Comment (george-bafaloukas-forgerock) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/FROptions.kt:150]: Will the non optional v alus allow the app to be used without setting an OAuth client at all? If they opt to use the FRSession classes, they can run the authentication without running the OAuth flows

Comment (jeyanthanperiyasamy) [reply forgerock-auth/src/main/java/org/forgerock/android/auth/FROptions.kt:150]: can we talk about this more, trying to understand this

Comment (jeyanthanperiyasamy) [reply forgerock-auth/src/main/java/org/forgerock/android/auth/FROptions.kt:150]: @george-bafaloukas-forgerock  . Both me and @witrisna  tested this flow for both static and dynamic config, we are able to get the SSO Token for FRsession.authenticate. Thanks for bringing this scenario to test

File: forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt (hunk 1 above)
Comment (jeyanthanperiyasamy) [reply forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt:20]: done

**Review by witrisna — APPROVED** [review]

**Pull request merged**